lib/matplotlib/__init__.py

```diff
@@ -1570,7 +1570,15 @@ def _init_tests():
         warnings.warn(
             "matplotlib is not built with the correct FreeType version to run "
             "tests.  Set local_freetype=True in setup.cfg and rebuild. "
-            "Expect many image comparison failures below.")
+            "Expect many image comparison failures below. "
+            "Expected freetype version {0}. "
+            "Found freetype version {1}."
```

Comment:
And, don't worry. We won't ask you to rebase/squash for this. It was only to clean out a messy revision history. Fixing one little thing doesn't need a rebase/squash in my opinion.